src/sentry/snuba/tasks.py

```diff
@@ -195,16 +188,6 @@ def delete_subscription_from_snuba(query_subscription_id, **kwargs):
         subscription.update(subscription_id=None)
 
 
-def build_query_builder(
-    entity_subscription: BaseEntitySubscription,
-    query: str,
-    project_ids: list[int],
-    environment: Environment | None,
-    params: ParamsType | None = None,
-) -> QueryBuilder:
-    return entity_subscription.build_query_builder(query, project_ids, environment, params)
```

Comment:
literally all this helper method is doing is invoking `build_query_builder` on the passed `entity_subscription` 🤔 

This helper is unnecessary and can be removed

Reply:
👍 makes sense to remove, especially with the entity_subscription being an argument 😅